.github/workflows/index.yml

```diff
@@ -37,11 +37,13 @@ jobs:
       - name: Install dependencies
```

Comment:
why this code block is repeated twice? I mean from line 22 to 27 and from 37 to 40;